[2024-01-31T18:34:14.375+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: user_automation.stream_data_from_api manual__2024-01-31T18:34:09.973814+00:00 [queued]>
[2024-01-31T18:34:14.389+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: user_automation.stream_data_from_api manual__2024-01-31T18:34:09.973814+00:00 [queued]>
[2024-01-31T18:34:14.391+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2024-01-31T18:34:14.421+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): stream_data_from_api> on 2024-01-31 18:34:09.973814+00:00
[2024-01-31T18:34:14.431+0000] {standard_task_runner.py:57} INFO - Started process 138 to run task
[2024-01-31T18:34:14.446+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'user_automation', 'stream_data_from_api', 'manual__2024-01-31T18:34:09.973814+00:00', '--job-id', '23', '--raw', '--subdir', 'DAGS_FOLDER/kafkastream.py', '--cfg-path', '/tmp/tmpzntufgyr']
[2024-01-31T18:34:14.455+0000] {standard_task_runner.py:85} INFO - Job 23: Subtask stream_data_from_api
[2024-01-31T18:34:14.741+0000] {task_command.py:415} INFO - Running <TaskInstance: user_automation.stream_data_from_api manual__2024-01-31T18:34:09.973814+00:00 [running]> on host 9654d1909d72
[2024-01-31T18:34:14.936+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='user_automation' AIRFLOW_CTX_TASK_ID='stream_data_from_api' AIRFLOW_CTX_EXECUTION_DATE='2024-01-31T18:34:09.973814+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-01-31T18:34:09.973814+00:00'
[2024-01-31T18:34:15.063+0000] {conn.py:380} INFO - <BrokerConnection node_id=bootstrap-0 host=broker:29092 <connecting> [IPv4 ('172.18.0.8', 29092)]>: connecting to broker:29092 [('172.18.0.8', 29092) IPv4]
[2024-01-31T18:34:15.065+0000] {conn.py:1205} INFO - Probing node bootstrap-0 broker version
[2024-01-31T18:34:15.068+0000] {conn.py:410} INFO - <BrokerConnection node_id=bootstrap-0 host=broker:29092 <connecting> [IPv4 ('172.18.0.8', 29092)]>: Connection complete.
[2024-01-31T18:34:15.180+0000] {conn.py:1267} INFO - Broker version identified as 2.5.0
[2024-01-31T18:34:15.182+0000] {conn.py:1268} INFO - Set configuration api_version=(2, 5, 0) to skip auto check_version requests on startup
[2024-01-31T18:34:16.192+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=broker:29092 <connecting> [IPv4 ('172.18.0.8', 29092)]>: connecting to broker:29092 [('172.18.0.8', 29092) IPv4]
[2024-01-31T18:34:16.193+0000] {conn.py:410} INFO - <BrokerConnection node_id=1 host=broker:29092 <connecting> [IPv4 ('172.18.0.8', 29092)]>: Connection complete.
[2024-01-31T18:34:16.194+0000] {conn.py:919} INFO - <BrokerConnection node_id=bootstrap-0 host=broker:29092 <connected> [IPv4 ('172.18.0.8', 29092)]>: Closing connection. 
[2024-01-31T18:34:48.268+0000] {kafkastream.py:57} ERROR - An error occured: HTTPSConnectionPool(host='randomuser.me', port=443): Max retries exceeded with url: /api/ (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1129)')))
[2024-01-31T18:35:16.050+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=broker:29092 <connected> [IPv4 ('172.18.0.8', 29092)]>: Closing connection. 
[2024-01-31T18:35:16.054+0000] {python.py:194} INFO - Done. Returned value was: None
[2024-01-31T18:35:16.089+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=user_automation, task_id=stream_data_from_api, execution_date=20240131T183409, start_date=20240131T183414, end_date=20240131T183516
[2024-01-31T18:35:16.148+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-01-31T18:35:16.184+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
